src/compiler/crystal/tools/init/template/readme.md.ecr

```diff
@@ -5,15 +5,15 @@ TODO: Write a description here
 ## Installation
 
 <%- if config.skeleton_type == "lib" -%>
-Add this to your application's `shard.yml`:
+1. Run `shards init`
+2. Add this to your application's `shard.yml`:
```

Comment:
Better to detail what "this" is:
> Add the dependency to your application's `shard.yml`

Reply:
Let's exclude "application's". Because a `shard.yml` doesn't necessarily has to be used for just one application. I could just init the `shard.yml` on the desktop and use it for multiple applications.

So "Add the dependency to your `shard.yml`:"